patches/chrome-browser-resources-settings-about_page-about_page.html.patch

```diff
@@ -11,3 +11,12 @@ index 99c0a1c19a17bcb93b4e180e629f0f786477a4f2..3a043b55610c14cb4eddd6282b3e8308
                    $i18n{learnMore}
                  </a>
                </div>
+@@ -158,7 +158,7 @@
+                 </a>
+               </div>
+ </if>
+-              <div class="secondary">$i18n{aboutBrowserVersion}</div>
```

Comment:
This can be achieved via template modification in `brave_settings_overrides.js`

Something like: 
```js
{
...
'settings-about-page': (templateContent) => {
  templateContent.querySelector('#updateStatusMessage .secondary).innerHTML = loadTimeData.getString('aboutBrowserVersion')
}
```

Reply:
@petemill Is https://github.com/brave/brave-core/pull/3828/files#diff-166e28418b661667403b64249ce36c3a what you had in mind?